compiler/resolution/visibleFunctions.cpp

```diff
@@ -291,6 +291,13 @@ void initTypeHelperNames() {
   typeHelperNames.insert(astr_initCopy);
   typeHelperNames.insert(astr_autoCopy);
   typeHelperNames.insert(astr("chpl__autoDestroy"));
+
+  internalTypeHelperNames.insert(astr_cast);
+  internalTypeHelperNames.insert(astr_defaultOf);
+  internalTypeHelperNames.insert(astrNew);
```

Comment:
As we discussed, I'm hoping you will check to see if we can remove new and cast from this list.